src/content/docs/software/dubicube/bootstrap-method-cubes.md

```diff
@@ -74,6 +74,8 @@ $$
    - The standard deviation of the bootstrap replications, which estimates
    the variability of the statistic.
 
+<img src="figures/bootstrapping.png" alt="Bootstrap resampling a data cube." width="60%">
```

Comment:
This image doesn't show up in the rendered site (since there is no source file), is it required?